llvm/lib/SandboxIR/SandboxIR.cpp

```diff
@@ -59,8 +59,95 @@ void sandboxir::Value::printAsOperandCommon(raw_ostream &OS) const {
     OS << "NULL ";
 }
 
+void sandboxir::Argument::printAsOperand(raw_ostream &OS) const {
```

Comment:
Do you need to prefix everything with `sandboxir::`? I hoped the `using namespace sandboxir;` was sufficient.

Reply:
Yeah these can be dropped, let me remove `using namespace llvm` and replace it with `using namespace llvm::sandboxir`.